src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/action_group/_create.py

```diff
@@ -175,6 +198,10 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="Indicates whether this instance is global runbook.",
             required=True,
         )
+        _element.managed_identity = AAZStrArg(
+            options=["managed-identity"],
+            help="The principal id of the managed identity. The value can be \"None\", \"SystemAssigned\" ",
+        )
```

Comment:
This need to be added into `--action` and exposed into users, or expose the whole parameter.